architecture/core_concepts/image_streams.adoc

```diff
@@ -0,0 +1,614 @@
+
+----
+oc describe is/python
+
+Name:			python
+Namespace:		default
+Created:		5 minutes ago
+Labels:			<none>
+Annotations:		openshift.io/image.dockerRepositoryCheck=2017-10-02T17:05:11Z
+Docker Pull Spec:	docker-registry.default.svc:5000/default/python
+Image Lookup:		local=false
+Unique Images:		1
+Tags:			2
+
+latest
+  tagged from python@sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+
+  * centos/python-35-centos7@sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+      About a minute ago
+
+3.5
+  tagged from centos/python-35-centos7
+
+  * centos/python-35-centos7@sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+      5 minutes ago
+----
+
+[[image-stream-mappings-working-adding-tags]]
+=== Adding Tags to an External Image
+
+use the `oc tag` command for all tag-related operations, such as adding tags pointing to internal or external images:
+
+----
+oc tag <repositiory/image> <image-name:tag>
+----
+
+For example, this command maps the `docker.io/python:3.6.0` image to the `3.6` tag in the `python` image stream. 
+
+----
+oc tag docker.io/python:3.6.0 python:3.6
+Tag python:3.6 set to docker.io/python:3.6.0.
+----
+
+If the external image is secured, you will need to create a secret with credentials for accessing that registry. 
+See xref:../../dev_guide/managing_images.adoc#private-registries[Importing Images from Private Registries] for more details.
+
+[[image-stream-mappings-working-updating]]
+=== Updating an Image Stream Tag
+
+To update the `latest` tag to reflect the newer `3.6` tag in our image stream:
+
+----
+oc tag <image-name:tag> <image-name:latest>
+----
+
+For example:
+
+----
+oc tag python:3.6 python:latest
+Tag python:latest set to python@sha256:438208801c4806548460b27bd1fbcb7bb188273d13871ab43f.
+----
+
+=== Removing Image Stream Tags from an Image Stream
+
+To remove old tags from an image stream:
+
+----
+oc tag -d <image-name:tag>
+----
+
+For example:
+
+----
+oc tag -d python:3.5
+
+Deleted tag default/python:3.5.
+----
+
+[[image-stream-mappings-working-periodic]]
+=== Configuring Periodic Importing of Tags
+
+When working with an external registry, to periodically re-import the image (to get latest security updates, and so forth), use the ` --scheduled` flag: 
+
+----
+oc tag <repositiory/image> <image-name:tag>  --scheduled
+----
+
+For example:
+
+----
+oc tag docker.io/python:3.6.0 python:3.6 --scheduled
+
+Tag python:3.6 set to import docker.io/python:3.6.0 periodically.
+----
+
+This command causes {product-title} to periodiallly update this particular image stream tag should. This period is a cluster-wide setting is set to 15 minutes by default.
+
+Only an image stream tag pointing to an external Docker registry can be periodically checked for updates.
+
+To remove the periodical check, re-run above command but omit the --scheduled flag. This will reset its behavior to default.
+
+----
+oc tag <repositiory/image> <image-name:tag> 
```

Comment:
Can we have this describe in only one document and all others will be only pointing to it? I'm confused now.

Reply:
@soltysh Not sure what you are asking. Do we want to move the "working with" concepts into the introductory sections? For ex. this command would go into the "Image Stream Tag" section?

Reply:
@soltysh Bump ^^